builtin/logical/transit/policy.go

```diff
@@ -25,6 +30,11 @@ const (
 	ErrTooOld = "ciphertext version is disallowed by policy (too old)"
 )
 
+const (
```

Comment:
Actually there's a cool optimization here. `iota` starts at zero unless you tell it not to, so if we simply remove `KDFMode` from the `Policy` struct entirely, the default of `KDF` will be zero -- the old value. So that will be implicitly backwards compatible with no upgrade needed.